### ISSUE TITLE
Sign up with kyc-chain

### ISSUE THREAD

**Issue by rodrigopavezi** [SelfKeyFoundation/Identity-Wallet#1097]

**Comment by sk91** [comment]
@rhdelima why is this qa blocked?

**Comment by rhdelima** [comment]
@sk91 ,
No available exchange to test this. All exchanges have Coming Soon and no link to open an exchange at least one. The last time I checked this ticket, there was no Sign Up button on the exchange screen.

**Comment by CortecKYC** [comment]
@rhdelima Can you double-check now whether this ticket can be QA'd? If so, please go ahead.

**Comment by rhdelima** [comment]
@CortecKYC I don't think this is still applicable. The sign up button on the exchanges contain the affiliate link.

**Comment by CortecKYC** [comment]
Oh yes, sorry. I read your last comment and I misunderstood the ticket. I'll add a quick description. 
Sorry @rhdelima
